CONTRIBUTING.md

```diff
@@ -12,19 +12,26 @@ team of web-focused volunteers that meet weekly to create projects like [Arizona
   * [Slack](https://uarizona.slack.com) is our main source of communications.
     * Use the `#ua-quickstart-d8` channel for questions/comments related to this
       project.
-    * Use the `#friday-meetings` channel to ask questions or get updates related 
+    * Use the `#friday-meetings` channel to ask questions or get updates related
       to Arizona Digital meetings, both physical and via Zoom.
     * Use the `#uadigital-general` channel to ask general questions related to
       Arizona Digital.
   * A basic understanding of [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-  * [Lando](https://docs.lando.dev/basics/installation.html) or [DDEV](https://www.ddev.com/get-started/) for local development.
+  * Local containerized (Docker) development environment tool either lando or ddev
+    * [Lando](https://docs.lando.dev/basics/installation.html)
+    * [DDev](https://docs.lando.dev/basics/installation.html)
+  * An IDE with the capability to attach to a remote codeserver or docker
+    container. [Visual Studio](https://visualstudio.microsoft.com) allows for this.
```

Comment:
Visual Studio Code

Reply:
Maybe https://code.visualstudio.com instead of https://visualstudio.microsoft.com ?

Reply:
I've updated, thanks for the link!